ZenPacks/zenoss/Microsoft/Windows/tests/testIISSiteDataSource.py

```diff
@@ -63,7 +63,8 @@ def test_onError_status(self):
 
         config = MagicMock(
             id='127.0.0.1',
-            datasources=[MagicMock(params={'eventlog': sentinel.eventlog})],
+            datasources=[MagicMock(params={'eventlog': sentinel.eventlog}, zWinRMKRBErrorThreshold=0), ],
+
```

Comment:
Just a nitpick: please remove the extra comma and space:
```
zWinRMKRBErrorThreshold=0), ],
                          ^^
```

Reply:
Done.